lib/js-api/src/api/gs-core-rest-api/api/legal-hold.api.ts

```diff
@@ -52,4 +53,64 @@ export class LegalHoldApi extends BaseApi {
             returnType: HoldPaging
         });
     }
+
+    /**
+     * Assign hold of legal holds
+     *
+     * @param holdId The identifier of a hold.
+     * @param ids one element list with id of hold to assign to existing hold
+     * @returns Promise<HoldEntry>
+     */
+    assignHold(ids: [string], holdId: string): Promise<HoldEntry> {
+        throwIfNotDefined(holdId, 'holdId');
+        throwIfNotDefined(ids, 'ids');
+
+        return this.post({
+            path: `/holds/{holdId}/children`,
+            pathParams: { holdId },
+            bodyParam: ids.map((id) => ({
+                id
+            })),
+            returnType: HoldEntry
+        });
+    }
+
+    /**
+     * Assign holds of legal holds
+     *
+     * @param holdId The identifier of a hold.
+     * @param ids list of ids of holds to assign to existing hold
+     * @returns Promise<HoldPaging>
+     */
+    assignHolds(ids: string[], holdId: string): Promise<HoldPaging> {
+        throwIfNotDefined(holdId, 'holdId');
+        throwIfNotDefined(ids, 'ids');
+
+        return this.post({
+            path: `/holds/{holdId}/children`,
+            pathParams: { holdId },
+            bodyParam: ids.map((id) => ({
+                id
+            })),
+            returnType: HoldPaging
+        });
+    }
+
+    /**
+     * Deletes the relationship between a child with id holdChildId and a parent hold with id holdId.
+     *
+     * @param holdId The handled hold Id
+     * @param holdChildId The Id of the child hold which is deleted
+     * @returns Empty response
+     */
+    unassignHold(holdId: string, holdChildId: string): Promise<undefined> {
+        throwIfNotDefined(holdId, 'holdId');
+        throwIfNotDefined(holdChildId, 'holdChildId');
+
+        return this.delete({
+            path: `/holds/{holdId}/children/{holdChildId}`,
+            pathParams: { holdId, holdChildId },
+            returnType: undefined
+        });
```

Comment:
I don't see `nodeId` here.
Our goal of this story is **assignment** and **unassignment** hold to the node.
This endpoint **delete** hold from Record Management at all. So no one will be able to select it again because there is no longer such a hold.
Please integrate correct endpoint to unassign hold from the node